src/cmd/ksh93/sh/path.c

```diff
@@ -47,10 +47,12 @@ static void funload(Shell_t *, int, const char *);
 static void exscript(Shell_t *, char *, char *[], char *const *);
 static bool path_chkpaths(Shell_t *, Pathcomp_t *, Pathcomp_t *, Pathcomp_t *, int);
 static void path_checkdup(Shell_t *shp, Pathcomp_t *);
+static Pathcomp_t *defpath_init(Shell_t *shp);
 
 static const char *std_path;
```

Comment:
Mind adding a `= NULL` here? I know it's not strictly speaking necessary since static vars are guaranteed to be initialized to zero but it's better to be explicit.